job_spec/ARIA_S1_GUNW.yml

```
@@ -47,7 +29,7 @@ ARIA_S1_GUNW:
     DEFAULT:
       cost: 1.0
   validators:
-    - check_dem_coverage
+    - check_aria_s1_gunw_dates
```

I assume want to keep the DEM coverage check?
```suggestion
    - check_aria_s1_gunw_dates
    - check_dem_coverage
```

---

Oh wait nevermind, I get it.